apps/core/components/sectorDetail/segment/SegmentConstants.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+
+export const mockTransporterData = [
+  {
+    title: 'Connection Info',
+    information: [
+      {
+        icon: 'airplane-down',
+        label: 'Flight No',
+        value: 'OE 570',
+      },
+      {
+        icon: 'boarding-gate',
+        label: 'Departure terminal',
+        value: 'B7',
+      },
+      {
+        icon: 'boarding-gate',
+        label: 'Destination Terminal',
+        value: 'A2',
+      },
+    ],
+  },
+  {
+    title: 'Seating Info',
+    information: [
+      {
+        icon: 'seat',
+        label: 'Seat pitch',
+        value: '36-38"',
+      },
+      {
+        icon: 'seat',
+        label: 'Seat width',
+        value: '17"',
+      },
+      {
+        icon: 'seat',
+        label: 'Seat recline',
+        value: '3"',
+      },
+      {
+        icon: 'entertainment',
+        label: 'Audio & video on demand',
+        value: 'Yes',
```

Comment:
Why not use `true`/`false` instead of `'Yes'`?

Reply:
Those data were basically prepared according to design https://www.figma.com/file/XD9kQTOJxaqviaxD0g3AhAXS/Booking-Detail .
And it's really just mock for now. We are currently not sure if we will be even able to dig this info from api and in which form. Maybe in this case it can also be something like `just audio` but we don't know yet.

Reply:
Hmm, true.., OK